### DISCUSSION TITLE
Welcome to MenuTube Discussions!

### DISCUSSION THREAD

**Discussion by edanchenkov** [edanchenkov/MenuTube#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ddenya** [comment]
1. Pls add ability to resize window so only a video frame and controls are visible on click.
https://ibb.co/VW3YjLP
2. Add feature to cycle the video (song in my case)

**Reply by edanchenkov** [reply]
Hello, thanks for reaching out!

Could you create 2 separate issues for your feature requests [here](https://github.com/edanchenkov/MenuTube/issues)?

Unfortunately, right now I have very little time to update MenuTube with new features. And even if I had time I would spend it on optimyzing existing build by updating core libraries for better new macOs support. 

All feedback and PR are greatly appreciated, I would love to see some code from you! 💪

Thanks for using MenuTube!

**Comment by ddenya** [comment]
Hello Egor,


> Could you create 2 separate issues for your feature requests here <https://github.com/edanchenkov/MenuTube/issues>?
 
This is done.

> All feedback and PR are greatly appreciated, I would love to see some code from you! 💪

No prob. Maybe one day. Your app is nice though.

**Comment by PxBCPG** [comment]
Hi!
Thanks for your app, which is the nicest I've ever seen in that domain! 
Any plan to do something for mobile or any other platform, or it's impossible to do it easily? 
Thank you again for this app,
BCPG

**Reply by edanchenkov** [reply]
@PxBCPG Hello! Thanks for using the app.

Unfortunately, it is not trivial to move this app to mobile as is, and specifically to keep some core functionality that makes this app valuable. I had some local experiments with Flutter application and used it in development mode for some time on my phone, but then it stopped working with some latest iOS updates.

Right now, I don't have time to fully support this app or build another one. Let's see how the situation unfolds in the future and probably I will make some important updates to MenuTube for MacOs in upcoming months. Mostly update dependencies, stability improvements and maybe even some functionality updates.

**Reply by PxBCPG** [reply]
super cool!
another idea will be to add youtube music support, but as you said, you don't always have the time...
